src/test/java/seedu/address/model/poc/NameTest.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.model.poc;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class NameTest {
```

Comment:
Might be better if we have a toString() method test as previously recommended in EmailTest